playbooks/webapp/tasks/create.yaml

```diff
@@ -23,8 +22,7 @@
 
     - name: List availability zones from aws region
       amazon.aws.aws_az_info:
-        filters:
-          region-name: "{{ region }}"
+        region: "{{ region }}"
```

Comment:
I guess this is equivalent to:

```
- name: List availability zones from aws region
  amazon.aws.aws_az_info:
```

Reply:
While I don't have a strong preference, I shall make the change.